packages/fx-core/src/component/generator/templates/templateNames.ts

```diff
@@ -3,11 +3,9 @@
 import { Inputs } from "@microsoft/teamsfx-api";
 import {
   ApiAuthOptions,
-  ApiPluginStartOptions,
```

Comment:
TODO: deprecate mapping in this file.

Reply:
Will be retired after new pattern is applied.